packages/grid/_modules_/grid/models/colDef/gridColDef.ts

```diff
@@ -38,12 +38,12 @@ export interface GridColDef {
    */
   width?: number;
   /**
-   * If set, it indicates that a column has fluid width. Range [0, ∞].
+   * If set, it indicates that a column has fluid width. Range [0, ∞).
```

Comment:
`[ ...  )`??

Reply:
If one end of the range is infinity the mathematical representation that needs to be used is `)` because it is not a hard end of the range 😉

Reply:
In the French system, it would have been `[0, ∞[`.